tests/jsmarionette/runner/marionette-js-runner/host/python/runner-service/runner_service/handlers/runner.py

```diff
@@ -16,6 +18,59 @@
 # We log over a special pipe (fd3) which node knows to hook up.
 GECKO_LOG_FD = 3
 
+class DeviceUtils:
+    def __init__(self, deviceManager):
+        self.dm = deviceManager
+
+    def _purge_data(self):
+        # Clean up data on device.
+        print("cleaning up data on device.")
```

Comment:
I feel like I ran into encoding issues v UTF-8 , ISO-8869-1 and UTF-16 when using ' for print() because it assumed it was a string encoded differently than the system locale. That's why print and Exception use " vs '